imaginairy/enhancers/upscale.py

```diff
@@ -28,7 +28,7 @@
 
 def upscale_image(
     img: LazyLoadingImage | Image.Image,
-    upscaler_model: str = "realesrgan",
+    upscaler_model: str,
```

Comment:
why was default removed

Reply:
Will add it back. Thought it was confusing to have default set in two places.

Should it be "realesrgan"?

Reply:
I created "DEFAULT_UPSCALE_MODEL" in config.py and use that for the two applicable places.